tests/benchmark/datasets.md

```diff
@@ -0,0 +1,106 @@
+# Datasets
+
+## Overview
+
+The following datasets are used for running performance benchmarking on Astro SDK `load_file` feature.
+
+All of them are based on real external datasets. For reproducibility of the benchmark tests, they were copied to:
+`gs://astro-sdk/benchmark/`
+
+Within `gs://astro-sdk/benchmark/`, there are two paths:
+* `original`: files with original size
+* `trimmed`: files trimmed to meet the desired sizes (for example, 10 GB)
+
+|            | size   | format | rows      | columns | path to trimmed file(s)                     | description                 |
+|------------|--------|--------|-----------|---------|---------------------------------------------|-----------------------------|
+| ten_kb     | 10 KB  | csv    | 158       | 8       | covid_overview/covid_overview_10kb.csv      | UK Covid overview sample    |
+| hundred_kb | 100 KB | csv    | 748       | 9       | tate_britain/artist_data_100kb.csv          | Tate Gallery artist sample  |
+| ten_mb     | 10 MB  | csv    | 600,000   | 3       | imdb/title_ratings_10mb.csv                 | IMDB title ratings sample   |
```

Comment:
Should we include parquet for one of the datasets?

Reply:
Great idea, @kaxil! I'll do it

Reply:
Addressed, @kaxil !